app/grandchallenge/cases/widgets.py

```diff
@@ -51,7 +50,7 @@ def __init__(
     ):
         widgets = (
             ImageSearchWidget(),
-            UserUploadMultipleWidget(),
+            UserUploadSingleWidget(),

```

Comment:
For images, we might need to keep the `UserUploadMultipleWidget`, but I'm not fully sure. @jmsmkn do you know?